core-library/src/main/java/org/silverpeas/core/contribution/content/form/record/GenericRecordSetManager.java

```diff
@@ -1228,4 +1276,14 @@ private void closeConnection(Connection con) {
           " rec, " +
           TEMPLATE_TABLE +
           " tpl WHERE tpl.templatename = ? AND rec.templateId = tpl.templateId AND tf.recordId = rec.recordId";
+
+  private static final String SELECT_TEMPLATE_RECORDS_BY_FIELDVALUE =
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)